config/.well-known/apple-config.json

```diff
@@ -0,0 +1,11 @@
+{
+  "applinks": {
+    "apps": [],
+    "details": [
+      {
+        "appID": "P7G764E7RR.com.destinyitemmanager.app",
```

Comment:
It'd be neat if we could handle beta links too

Reply:
Hmm, I think that if we deploy this to beta.destinyitemmanager.com (which I think it will) — it should also capture beta links into the app. The appID here is just `<team id>.<app bundle id>`

Reply:
Yeah I think the app manifest has to claim which domains it handles though.